readme/Footer.scalatex

```diff
@@ -89,6 +89,10 @@
         Seems Twitter @lnk("now uses Ammonite",
         "https://twitter.com/bd/status/724297737525596160") within their
         monorepo (monorepi?) codebase
+      @li
+        Yang Bo from ThoughtWorks creates
```

Comment:
`s/creates/created/`?
```suggestion
        Yang Bo from ThoughtWorks created
```

Reply:
Thank you for figuring out the grammar error @alexarchambault